docs/README.md

```diff
@@ -127,7 +127,8 @@ positional arguments:
                         benchmark description file, or an openml suite or
                         task. OpenML references should be formatted as
                         'openml/s/X' and 'openml/t/Y', for studies and tasks
-                        respectively. Defaults to `test`.
+                        respectively. Use 'test.openml/s/X' for the test
```

Comment:
You changed this to `openmltestserver`, as far as I can tell. I would personally prefer `test.openml` but if that gave issues this works for me (but the docs need to be updated).

Reply:
you mean it should be changed in `openml.py`, right? the doc looks good to me, I also prefer `test.openml` over `openmltestserver` as it's easier to convert into a url:

`test.openml/t/4242` -> `test.openml.org/t/4242`

Reply:
Changed.